src/test/java/lotto_step2/LottoGameTest.java

```diff
@@ -24,12 +24,14 @@
     private InputView inputView = new InputView();
     private Lotto lotto;
     private LottoGame lottoGame;
+    private int bonus;
 
     @BeforeEach
     void init() {
         List<Integer> lastWeekLotto = new ArrayList<>(Arrays.asList(1, 2, 3, 4, 5, 6));
```

Comment:
```suggestion
        List<Integer> lastWeekLotto = Arrays.asList(1, 2, 3, 4, 5, 6);
```
아래 Arrays.asList 메소드 참고 부탁드립니다~

```java
    @SafeVarargs
    @SuppressWarnings("varargs")
    public static <T> List<T> asList(T... a) {
        return new ArrayList<>(a);
    }
```

Reply:
앗 예제 코드까지 감사합니다.